ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -619,16 +619,16 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: baremetal-lab-upi
-- as: gcp-ipi-oidc-rt-fips-amd-f14
-  cron: 25 18 1,17 * *
-  steps:
-    cluster_profile: gcp-qe
-    env:
-      EXTRACT_MANIFEST_INCLUDED: "true"
-      FIPS_ENABLED: "true"
-      RT_ENABLED: "true"
-    test:
-    - chain: openshift-upgrade-qe-test
+- as: gcp-ipi-oidc-rt-fips-amd-f14	
+  cron: 25 18 1,17 * *	
+  steps:	
+    cluster_profile: gcp-qe	
+    env:	
+      EXTRACT_MANIFEST_INCLUDED: "true"	
+      FIPS_ENABLED: "true"	
+      RT_ENABLED: "true"	
+    test:	
+    - chain: openshift-upgrade-qe-test	
```

Comment:
It seems there are many extra spaces in the job, @jianlinliu can you confirm?
BTW, if you objection, can we use f28 for all upgrade jobs?

Reply:
I am saying the job is being edited in your PR, I will leave this job untouched in my PR.